rasa/core/policies/policy.py

```diff
@@ -134,24 +136,31 @@ def featurize_for_training(
             training_trackers:
                 the list of the :class:`rasa.core.trackers.DialogueStateTracker`
             domain: the :class:`rasa.core.domain.Domain`
+            interpreter: the :class:`rasa.core.interpreter.NaturalLanguageInterpreter`
 
         Returns:
-            the :class:`rasa.core.training.data.DialogueTrainingData`
+            - a dictionary of attribute (INTENT, TEXT, ACTION_NAME, ACTION_TEXT,
+              ENTITIES, SLOTS, FORM) to a list of features for all dialogue turns in
+              all training trackers
+            - the label ids (e.g. action ids) for every dialuge turn in all training
+              trackers
         """
 
-        training_data = self.featurizer.featurize_trackers(
+        X, label_ids = self.featurizer.featurize_trackers(
             training_trackers, domain, interpreter
         )
 
+        # TODO: does this still fit?
```

Comment:
@Ghostvv Does it make sense to just take the first x features from the list of list of list of features? Or do we need to change the logic here?

Reply:
what do you mean? we take first x examples

Reply:
was it also a list of lists before? if not we might need to change the logic. just wanted to double check.

Reply:
before it was numpy ndarray, but the outer dimension is always batch dimension or examples dimension

Reply:
ok, then I guess we should be fine. thanks.